libs/iota-browser/src/helpers/response-handler.ts

```diff
@@ -18,7 +16,7 @@ import { Logger } from '@affinidi-tdk/common/helpers'
 
 export type IotaResponse = {
   correlationId: string
-  vpToken: VerifiablePresentation
+  vpToken: any
```

Comment:
help me understand why it cannot be typed ? thanks

Reply:
I was gonna leave the same comment, also curious about this 👍

Reply:
It could be, but out of scope right now. The `VerifiablePresentation` type we had here is incomplete, which introduced the issue. This is a hotfix.

TDK doesn't use these objects, it just returns them as is. In a follow up ticket we can add proper validation of the received VP token and for the presentation submission, but we will need to diligently craft those types/validators.

Reply:
does it work:
```typescript
type VerifiablePresentation = any 
```
?

Reply:
One thing we can do is to check why zod removed the additional fields when parsing. And progressively improve the zod schema. Let me try that

Reply:
Actually for the stuff I'm working on in Portal it would also be nice if we did have a complete `VerifiablePresentation` type